modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/DropZoneCommand.java

```diff
@@ -67,7 +67,8 @@ public List<UpdateEntry> get(Catalog catalog) {
                 .filter(t -> t.zoneId() == zone.id())
                 .findAny()
                 .ifPresent(t -> {
-                    throw new DistributionZoneBindTableException(zone.name(), t.name());
+                    throw new DistributionZoneCantBeDroppedValidationException("Distribution zone is assigned to the table [zoneName={}, "
```

Comment:
i`d like to change sequence in message like : to the table [tableName={}, zoneName={}] i think it become more readable, but it up to you.

Reply:
Fixed.